tutorial/functions.py

```diff
@@ -17,18 +17,18 @@ def __init__(self, title=""):
         q2 = Question(
             question="What is the purpose of type hints in Python function definitions?",
             options={
-                "To enforce strict typing": "",
-                "To make the code more readable": "",
-                "To document the expected types": "Correct!",
+                "To enforce strict typing": "Does the interpreter enforce type hints?",
+                "Only to make the code more readable": "Not only...",
+                "To document the expected types of function arguments and return values": "Correct!",
             },
-            correct_answer="To document the expected types",
+            correct_answer="To document the expected types of function arguments and return values",
             shuffle=True,
         )
 
         q3 = Question(
-            question="What is the syntax to define a function that takes an arbitrary number of arguments in Python?",
+            question="What is the syntax to define a function with an arbitrary number of any type of argument?",
```

Comment:
How about:

Choose a function that accepts an arbitrary number of arguments of any type (positional and keyword).